.github/workflows/cherry-pick-since-release-branch.yml

```diff
@@ -55,7 +55,7 @@ jobs:
             for branch in "${target_branches[@]}"; do
               echo "$branch"
             done
-            echo "branches=$(printf '%s\n' "${target_branches[@]}" | jq -R . | jq -s .)" >> "$GITHUB_OUTPUT"
+            echo "branches=$(printf '%s\n' "${target_branches[@]}" | jq -R . | jq -s -c .)" >> "$GITHUB_OUTPUT"
```

Comment:
jq compact output is also used by the single-cherry-pick action:
https://github.com/risingwavelabs/risingwave/blob/9dfd3a69f9a913b8d9d5b4cc1feb40a7ea4cc8f9/.github/workflows/cherry-pick-to-release-branch.yml#L25

error:
https://github.com/risingwavelabs/risingwave/actions/runs/13936910765/job/39006484878
> Matching release branches found:
> release-2.1
> release-2.2
> release-2.3
> Error: Unable to process file command ‘output’ successfully.
> Error: Invalid format '  “release-2.1”,'

Alternative ways (`<<` heredoc or `tr`) to handle multiline:
https://stackoverflow.com/questions/74137120/how-to-fix-or-avoid-error-unable-to-process-file-command-output-successfully